src/pytest_cov/engine.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, cov_source, cov_report, cov_config, cov_append, config=None,
         """Get some common config used by multiple derived classes."""
         self.cov_source = cov_source
         self.cov_report = cov_report
-        self.cov_config = os.path.abspath(cov_config)
+        self.cov_config = os.path.abspath(cov_config) if os.path.exists(cov_config) else cov_config
```

Comment:
Actually, I'm sure it _was_ intended for this, at least in part. I just wonder if this is more of a design bug in coverage. It's also something we want to address here, as usage in tox is very common. Still, it does seem cumbersome.